docs/products/clickhouse/get-started.md

```diff
@@ -5,77 +5,486 @@ sidebar_label: Get started
+[Aiven Console](https://console.aiven.io).
+</TabItem>
+<TabItem value="2" label="Terraform">
+
+1. [Create an authentication token](/docs/platform/howto/create_authentication_token).
+
+1. Create the `sample.tf` file for the `aiven` provider configuration and
+   the `aiven_clickhouse` resource.
+
+    ```hcl
+    variable "aiven_token" {
+      type = string
+    }
+
+    variable "aiven_project_name" {
+      type = string
+    }
+
+    terraform {
+      required_providers {
+        aiven = {
+          source  = "aiven/aiven"
+          version = ">=4.0.0, <5.0.0"
+        }
+      }
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_token
+    }
+
+    resource "aiven_clickhouse" "clickhouse" {
+      project                 = var.aiven_project_name
+      cloud_name              = "google-europe-west1"
+      plan                    = "startup-16"
+      service_name            = "my-clickhouse"
+      maintenance_window_dow  = "friday"
+      maintenance_window_time = "23:00:00"
+
+      clickhouse_user_config {
+        service_log = false
+      }
+    }
+
+    output "clickhouse_service_host" {
+      value = aiven_clickhouse.clickhouse.service_host
+    }
+    output "clickhouse_service_port" {
+      value = aiven_clickhouse.clickhouse.service_port
+    }
+    output "clickhouse_service_username" {
+      value = aiven_clickhouse.clickhouse.service_username
+    }
+    output "clickhouse_service_password" {
+      value     = aiven_clickhouse.clickhouse.service_password
+      sensitive = true
+    }
+    ```
+
+1. Create the `terraform.tfvars` file for assigning actual values to your previously
+   declared variables.
+
+   ```hcl
+   aiven_token    = "AIVEN_TOKEN"
+   aiven_project_name = "PROJECT_NAME"
+   ```
+
+1. Run `terraform init` > `terraform plan` > `terraform apply --auto-approve`.
+
+1. Store Terraform outputs in environment variables so that they can be used for
+   [connecting](#connect-to-service):
+
+   ```bash
+   CLICKHOUSE_HOST="$(terraform output -raw clickhouse_service_host)"
+   CLICKHOUSE_PORT="$(terraform output -raw clickhouse_service_port)"
+   CLICKHOUSE_USER="$(terraform output -raw clickhouse_service_username)"
+   CLICKHOUSE_PASSWORD="$(terraform output -raw clickhouse_service_password)"
+   ```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+
+Create an Aiven for ClickHouse service using the Aiven Operator for Kubernetes.
+
+1. [Get authenticated and authorized](https://aiven.github.io/aiven-operator/authentication.html).
+1. Create file `example.yaml` with the following content:
+
+   ```yaml
+   apiVersion: aiven.io/v1alpha1
+   kind: Clickhouse
+   metadata:
+     name: my-clickhouse
+   spec:
+     authSecretRef:
+       name: aiven-token
+       key: token
+
+     connInfoSecretTarget:
+       name: my-clickhouse-connection
+       annotations:
+         foo: bar
+       labels:
+         baz: egg
```

Comment:
I would remove all the annotations / labels / tags, they are just providing noise IMO.